rustworkx-core/src/connectivity/mod.rs

```diff
@@ -22,4 +22,6 @@ pub use biconnected::articulation_points;
 pub use chain::chain_decomposition;
 pub use conn_components::bfs_undirected;
 pub use conn_components::connected_components;
+pub use conn_components::is_connected;
+// pub use conn_components::node_connected_components;
```

Comment:
```suggestion
pub use conn_components::node_connected_component;
```
This needs to be here. There was just a typo.

Reply:
I get an unresolved import error. It can't seem to find `node_connected_components`